spec/models/spree/payment_spec.rb

```diff
@@ -27,7 +27,7 @@
       double('success_response', success?: true,
                                  authorization: '123',
                                  avs_result: { 'code' => 'avs-code' },
-                                 cvv_result: { 'code' => 'cvv-code', 'message' => "CVV Result" })
+                                 cvv_result: nil)
```

Comment:
A `nil` `cvv_result` indicates that additional authorization is not required.

Reply:
I think it might be `cvv_result: { code: nil, message: nil }` in that case (eg: the `code` and `message` keys are still present, but with `nil` values).